cli/cmd/alert_list.go

```diff
@@ -242,17 +251,29 @@ func listAlert(_ *cobra.Command, _ []string) error {
 		return errors.Wrap(err, msg)
 	}
 
+	// filter fixable
+	alerts := listResponse.Data
+	if alertCmdState.Fixable {
+		alerts, err = filterFixableAlerts(alerts, getRemediationTemplateIDs)
+		if err != nil {
+			return errors.Wrap(err, "unable to filter by alert fixability")
+		}
+	}
+
 	if cli.JSONOutput() {
-		return cli.OutputJSON(listResponse.Data)
+		return cli.OutputJSON(alerts)
 	}
 
-	if len(listResponse.Data) == 0 {
+	if len(alerts) == 0 {
 		cli.OutputHuman("There are no alerts in your account in the specified time range.\n")
```

Comment:
If there are Alerts in the specified time range but the user provides the`--fixable` flag then this message is not accurate. Can we check if there are no alerts and the Fixable was set, then output a message like `There are no fixable alerts in the specified time range.`

Reply:
We can do some conditional messaging here, sure thing...

Reply:
Although, we could say the same thing about any filters (i.e. `--severity critical`).  There are alerts in the specified time range, but they were filtered out based on filter criteria.  I think I might find some more generic messaging to represent this situation.

Reply:
Fixed.